controllers/projectController.js

```diff
@@ -3,15 +3,20 @@ const ProjectModel = require('../models/projectModel');
 const UserModel = require('../models/userModel');
 const CommentModel = require('../models/commentModel');
 const ContributorModel = require('../models/contributorModel');
-const ContributorRelationships = require('../models/contributorsRelationship');
-
+const ContributorRelationshipsModel = require('../models/contributorsRelationship');
+const ProjectsRelationshipModel = require('../models/projectsRelationship');
 const ProjectValidationSchema = require('../validations/projectValidation');
+const { findOne } = require('../models/projectModel');
 
 const controller = {
   showAllProjects: async (req, res) => {
     let projects = [];
     try {
-      projects = await ProjectModel.aggregate([{ $match: {} }, { $sort: { updatedAt: -1 } }]);
+      projects = await ProjectModel.aggregate([
+        { $match: {} },
+        { $project: { _id: 0, user_id: 0, description: 0 } },
+        { $sort: { updatedAt: -1 } },
+      ]);
     } catch (error) {
       res.status(500);
```

Comment:
do we need to send back an error message (eg "failed to get projects list") with this res.status?